app/parser.py

```diff
@@ -0,0 +1,35 @@
+from app.database import get_database
+import click
+
+# parses a .csv that is passed to it.  Creates an array with
+# the data
+def parse(file):
+    file_contents_by_row = []
+    last_line = None
+    with open(file) as sample_file:
+        for line in sample_file:
+            if not last_line:
+                row = line.split(",")
+                file_contents_by_row.append(row)
+    if len(file_contents_by_row[0]) is not 13:
+        click.echo('ERROR: csv is not in correct format')
+        return
+    fill_database(file_contents_by_row)
+
+
+# fills the database with an array that is passed with it
+def fill_database(file_contents_by_row):
+
+    #use this a placeholder for the crn
+    crn = 0
+
+    db = get_database()
+
+    for x in file_contents_by_row[4:1231]:  # 1231 is the number of rows that got parsed into the array
```

Comment:
We should track how many rows are in the files when we read it in here, so we don't have to hard code the number of lines in the file. Also I think you can do something like ```file_contents_by_row[4:]``` which should just go from line 4 to the end of the array

Reply:
I must've missed this one.  I didn't know about that when I made my first parser and thought I got rid of the hard coded result and then just copied that over.